docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -39,342 +37,123 @@ Devices with Intel® GNA support:
+### Automatic QoS Feature on Windows*
 
-* Linux\* OS:
-[Download Intel® GNA driver for Ubuntu Linux 18.04.3 LTS (with HWE Kernel version 5.4+)](https://storage.openvinotoolkit.org/drivers/gna/)
+Starting with 2021.4.1 release of OpenVINO and 03.00.00.1363 version of Windows* GNA driver, a new execution mode `ov::intel_gna::ExecutionMode::HW_WITH_SW_FBACK` is introduced
+to assure that workloads satisfy real-time execution. In this mode, the GNA driver automatically falls back on CPU for a particular infer request
+if the HW queue is not empty, so there is no need for explicitly switching between GNA and CPU.
 
-* Windows\* OS:
-Intel® GNA driver for Windows is available through Windows Update\*
+@sphinxdirective
+.. tab:: C++
 
-## <a name="models-and-layers-limitations">Models and Layers Limitations</a>
+   .. code-block:: cpp
 
-Because of specifics of hardware architecture, Intel® GNA supports a limited set of layers, their kinds and combinations.
-For example, you should not expect the GNA Plugin to be able to run computer vision models, except those specifically adapted for the GNA Plugin, because the plugin does not fully support 2D convolutions.
+      ov::Core core;
+      model = core.read_model(path_to_model);
+      compiled_model = core.compile_model(model, "GNA", ov::intel_gna::execution_mode(ov::intel_gna::ExecutionMode::HW_WITH_SW_FBACK));
 
-For the list of supported layers, see the **GNA** column of the **Supported Layers** section in [Supported Devices](Supported_Devices.md).
+.. tab:: Python
 
-Limitations include:
+   .. code-block:: python
 
-- Only 1D convolutions are natively supported on the HW prior to GNA 3.0; 2D convolutions have specific limitations (see the table above).
-- The number of output channels for convolutions must be a multiple of 4.
-- The maximum number of filters is 65532 for GNA 2.0 and 8192 for GNA 3.0.
-- Transpose layer support is limited to the cases where no data reordering is needed or when reordering is happening for two dimensions, at least one of which is not greater than 8.
-- Splits and concatenations are supported for continuous portions of memory (e.g., split of 1,2,3,4 to 1,1,3,4 and 1,1,3,4 or concats of 1,2,3,4 and 1,2,3,5 to 2,2,3,4).
-- For Multiply, Add and Subtract layers, auto broadcasting is only supported for constant inputs.
+      from openvino.runtime import Core
+      core = Core()
+      model = core.read_model(model=path_to_model)
+      compiled_model = core.compile_model(model, device_name="GNA", config={'GNA_DEVICE_MODE' : 'GNA_HW_WITH_SW_FBACK'})
 
-### Support for 2D Convolutions in Previous Generations of GNA Hardware
+@endsphinxdirective
 
-The Intel® GNA 1.0 and 2.0 hardware natively supports only 1D convolutions.
+> **NOTE**: Due to the "first come - first served" nature of GNA driver and the QoS feature, this mode may lead to increased CPU consumption
+if there are several clients using GNA simultaneously.
```

Comment:
this sentence starts somewhat weirdly, like something is missing.

Reply:
@yury-gorbachev This is a note on the text above.